.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": ["eslint:recommended", "plugin:react/recommended", "prettier"],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 12,
+    "sourceType": "module"
+  },
+  "plugins": ["react", "jsx-a11y", "prettier"],
```

Comment:
I noticed `jsx-a11y` was a thing we were expecting in `AcceptTermsPrompt`. There are likely more plugins we'll add here too.